include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -134,15 +134,26 @@ Operator<dim, Number>::distribute_dofs()
 
   // affine constraints
   affine_constraints.clear();
+  if(param.large_deformation)
```

Comment:
I think we need this for linear problems calling `rhs()/rhs_add()` as well, right?